src/components/core/wizard/desktop-wizard/desktop-wizard.tsx

```diff
@@ -146,22 +146,23 @@ const DesktopWizard = (props: DesktopWizard) => {
     const steps_config = steps.map((_step) => getStepDetails(_step));
     const current_step = steps[current_step_index];
 
+    const { isDark } = useTheme();
+
     return (
         <>
-            <LeftPanel dark={dark}>
+            <LeftPanel dark={isDark}>
```

Comment:
@michio-rework Just a question, Why we are using `variants` instead of `theme` itself in `styled-components`? 🤔

Reply:
in this project we use stitches instead of styled-components and as of now the dark/light theme handling has been handled like so, we can actually update the components more to handle it the way you mentioned but in some cases we have to implement it like so. 
the changes in this PR are already huge enough and I can't put more stuff in it. 
can we discuss this with @balakrishna-binary and @yashim-deriv ? 
I really like your idea. it will give us flexibility in the future.

Reply:
Thanks for the explanation buddy 🙏🏻 

DX-wise, The developer shouldn’t care about if the theme is dark or like (except in some special cases) and that’s why we have theming. It seems a bit redundant to get the theme mode in each component and then passing it to styled component to set a color for example, We should be able to just switch the theme where the styled component has been provided and it should affect everywhere without needing to handle it manually.

Obviously, I'm not familiar with this project codebase and I'm just thinking out loud 😅 Please do ignore my comment 🙇🏻

Reply:
@farzin-fs `LeftPanel` component is not exposed outside and there is no way we can change the theme of this individual component. We pass dark prop to stitches to change the background Icon.